seed_files/SQL/tgencounterstat.sql

```diff
@@ -0,0 +1,18 @@
+use capiv2;
+
+truncate table tgencounterstat;
+
+load data infile '/srv/CAPI/CAPIv2/seed_files/CSV/tgencounterstat.csv' 
```

Comment:
Hard coded pathing, is this just for quick import? If so we should look at an alternative to allow for other quick seed file imports

Reply:
Ill change it to /tmp